index.html

```diff
@@ -0,0 +1,316 @@
+                        view.
+                        <div class="new-line"></div> “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+                        <div class="new-line"></div>
+                </div>
+
+            </section>
+        </article>
+
+
+
+    </main>
+    <footer class="flex">
+        <div id="main-footer-div" class="flex">
+            <div>
+                <ul>
+                    <li>Home Page</li>
+                    <li><a href="#" class="footer-link">World</a></li>
+                    <li><a href="#" class="footer-link">Coronavirus</a></li>
+                    <li><a href="#" class="footer-link">U.S.</a></li>
+                    <li><a href="#" class="footer-link">Politics</a></li>
+                    <li><a href="#" class="footer-link">Election Results</a></li>
+                    <li><a href="#" class="footer-link">New York</a></li>
+                    <li><a href="#" class="footer-link">Business</a></li>
+                    <li><a href="#" class="footer-link">Tech</a></li>
+                    <li><a href="#" class="footer-link">Science</a></li>
+                    <li><a href="#" class="footer-link">Sports</a></li>
+                    <li><a href="#" class="footer-link">Obituaries</a></li>
+                    <li><a href="#" class="footer-link">Today's Paper</a></li>
+                    <li><a href="#" class="footer-link">Corrections</a></li>
+                </ul>
+            </div>
+            <div>
+                <ul>
+                    <li>Opinion</li>
+                    <li><a href="#" class="footer-link">Today's Opinion</a></li>
+                    <li><a href="#" class="footer-link">Op-Ed Columnists</a></li>
+                    <li><a href="#" class="footer-link">Editorials</a></li>
+                    <li><a href="#" class="footer-link">Op-Ed Contributors</a></li>
+                    <li><a href="#" class="footer-link">Letters</a></li>
+                    <li><a href="#" class="footer-link">Sunday Review</a></li>
+                    <li><a href="#" class="footer-link">Video: Opinion</a></li>
+                </ul>
+            </div>
+            <div>
+                <ul>
+                    <li>Arts</li>
+                    <li><a href="#" class="footer-link">Today's Arts</a></li>
+                    <li><a href="#" class="footer-link">Art & Design</a></li>
+                    <li><a href="#" class="footer-link">Books</a></li>
+                    <li><a href="#" class="footer-link">Dance</a></li>
+                    <li><a href="#" class="footer-link">Movies</a></li>
+                    <li><a href="#" class="footer-link">Music</a></li>
+                    <li><a href="#" class="footer-link">Pop Culture</a></li>
+                    <li><a href="#" class="footer-link">Television</a></li>
```

Comment:
### Imrove the `Footer` Section

- [ ] A closer look at the footer, we can see the contrast. On the Source page, we have the page's logo and contents are divided into visible borders. Do you mind improving these areas?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/igorkol91/NYTarticle/newbranch/index.html)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/112322703-2fe4cd00-8cc2-11eb-89cd-6e9753263bdf.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/112322722-3410ea80-8cc2-11eb-941a-d59cdb637f4c.png)|